### PR TITLE
Upgrade the child killings

### PR DESCRIPTION
**Pull request by sbv-csis**
https://trello.com/c/b24IorU2

### CODE REVIEW

**Comment by car-csis** [comment]
`_timeout_signal1` ... `3` could perhaps be named better. Other than that I find the subject enjoyable. :+1:

File: src/proccer/agent.py [hunk 1]
```diff
@@ -90,11 +90,28 @@ def run_process(conf, name):
 def _in_parent(name, desc, pid, logfile):
     # Install signal handlers
     def _signalled(signo, frames):
-        log.debug('received signal %s, passing it on to child',
-                  signal_name.get(signo, str(signo)))
+        log.debug('%s(%d) received signal %s, passing it on to child',
+                  name, pid, signal_name.get(signo, str(signo)))
+        os.killpg(pid, signo)
+
+    def _timeout_signal1(_, __):
```
Comment (ski-csis) [inline src/proccer/agent.py]: Maybe soft, hard and hardest? 1, 2, 3 seems so boring... :P

**Comment by ski-csis** [comment]
Needs a new version too.

**Comment by ski-csis** [comment]
Other than that :+1:

**Comment by car-csis** [comment]
What about that version bumb Sune talked about?

**Comment by sbv-csis** [comment]
will do. I got a couple of more things I wanna bundle before releasing

**Comment by sbv-csis** [comment]
specifically around the `exec ` part of my tests. That needs to be global for this timeout to be really good

**Pull request merged**